File: exercicios_py/secao8/exercicio3.py
#Faça programa q carregue vetor com 10 numeros inteiros, mostrar na ordem inversa q foi digitado.

vetor = []
for i in range (0, 10):
    i = int(input("Digite um valor para o vetor:"))
    vetor.append(i)
print(vetor[9:-1:-1])

File: exercicios_java/exercicios_java/src/secao8/exercicio3.java
//Faça programa q carregue vetor com 10 numeros 
//inteiros, mostrar na ordem inversa q foi digitado.

package secao8;
import java.util.Scanner;

public class exercicio3 {
    public static void main(String[] args){
        int vetor[] = new int[10];
        Scanner t = new Scanner(System.in);

        for (int i = 0; i<10; i++){
            System.out.println("Informe o valor para o vetor: ");
            vetor[i] = t.nextInt();
        }for (int i = (vetor.length - 1); i>= 0; i--){
            System.out.println(vetor[i]);
        }
        t.close(); 
    }
}
